pkg/cmd/build/errors.go

```diff
@@ -32,7 +32,7 @@ func getErrorMessage(err error, tag string) error {
 	switch {
 	case isLoggedIntoRegistryButDontHavePermissions(err):
 		err = oktetoErrors.UserError{
-			E:    fmt.Errorf("error building image '%s': You are not authorized to push image '%s'", tag, imageTag),
+			E:    fmt.Errorf("error building image '%s': Access Denied (the repository does not exist or you are not authorized to push a new image).", imageTag),
```

Comment:
NIT: Maybe we should create an error for `Access Denied (the repository does not exist or you are not authorized to push a new image).` and check with `errors.Is()` in `remote.go` to return the proper message.
```go
accessDeniedErr = errors.New("Access Denied (the repository does not exist or you are not authorized to push a new image).")
err = oktetoErrors.UserError{
			E:    fmt.Errorf("error building image '%s': %w", imageTag, accessDeniedErr),
```

This code is shared with image builds and since the remote is not pushing any image can result in misleading error text. WDYT?

Reply:
@jLopezbarb I've updated the PR according to our chat :) Please have another look 🙏